sycl/plugins/level_zero/pi_level_zero.cpp

```diff
@@ -843,20 +843,92 @@ static const int ZeMaxCommandListCacheSize = [] {
   return CommandListCacheSizeValue;
 }();
 
-static const pi_uint32 ZeCommandListBatchSize = [] {
```

Comment:
I think the change deserves a unit test. Can we add it?

Reply:
We have tests for generic (documented) use of this option. I'd not spend time testing functionality of experimental controls. It's not set in stone, and is subject to change.